Methods/Problem13/Program.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System.Text;
+
+namespace Problem13
+{
+    internal class Program
+    {
+        static void Main(string[] args)
```

Comment:
Is all that logic necessary? 

Try having two Read Line for first polynom and next polynom and create (array of the numbers {-5, 1, 3}) like explained in Solutions and Guidelines Section. Try to find some rules and use them in the functions.